src/plugins/npm/npmPlugin.js

```diff
@@ -40,13 +35,13 @@ module.exports = async function npmPlugin(msg) {
           .slice(0, 5)
           .map((p) =>
             [
-              `npm.im/${p.name}`,
+              `npm.im/${p.name}`, // we could put the source link rather, using https://api.npms.io/v2/package/${p.name}
               p.version,
               p.date && p.date.slice(0, 10),
               getDesc(p.description, 80),
             ].join('|'),
           )
-          .join('\n'),
+          .join('  \n'),
```

Comment:
why this change? isn't this just adding trailing spaces?

Reply:
just adding more separation between each chunk of search results (in search mode, when command starts with ~), else they're all mixed together hard to read

Reply:
Maybe instead of joining it, we could keep it as an array the whole time?

Reply:
well, we need the response to be a string right?

Reply:
Sure but it doesn’t arrive as newlines, does it? So something must be changing that later?

Reply:
indeed, newlines seem to be lost here https://github.com/ljharb/jellobot/blob/main/src/processMessage.js#L23 
ok, let's drop the new lines completely there so, just spaces? I'll rebase edit that commit if you agree

Reply:
is it better now? using '  ⸺ ' as separator

Reply:
sounds great